doc/AST_FORMAT.md

```diff
@@ -997,6 +997,17 @@ Format:
  ~~ expression
 ~~~
 
+### "Excessed comma"
+
+Representation of the trailing comma after block arguments.
+
+~~~
+(procarg0)
+"|a,|"
+ ~~~~ expression
```

Comment:
No no, the expression for `(args)` encompasses `|a|` whereas the expression for `(args.arg)` encompasses `a`.